src/App.css

```diff
@@ -1,4 +1,116 @@
-.App {
-  text-align: center;
+@import url("https://artsyui.netlify.app/src/artsyui.css");
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;500;600&display=swap");
```

Comment:
suggestion: all the fa icons are available in **react-icons** so avoid duplication and reduce bundle size